src/main/kotlin/no/nav/syfo/dialogmote/database/MotedeltakerQuery.kt

```diff
@@ -91,6 +91,41 @@ fun DatabaseInterface.getMotedeltakerArbeidstakerById(motedeltakerId: Int): PMot
     return pMotedeltakerArbeidstakerList.first()
 }
 
+const val queryGetMotedeltakerArbeidstakerByIdent =
+    """
+        SELECT *
+        FROM MOTEDELTAKER_ARBEIDSTAKER
+        WHERE personident = ?
+    """
```

Comment:
Det viste seg vanskeligere enn antatt å bruke `WHERE personident IN ?` og sende inn en stringified versjon av lista. Ved bruk av `list.joinToString(',')` så fikk jeg jo bare `("fnr1, fnr2")`, mens det SQL ville ha var `("fnr1", "fnr")`. Det ble alltid sett på som én string, mens det egentlig er flere strings i SQLen

Reply:
Bør funke å gjøre map s -> ("'"+s+"'") før du gjør joinToString

Tror postgres foretrekker ('fnr1','fnr2')

Reply:
Jeg prøvde noe lignende, og problemet jeg møter er at den oppfatter alt (også kommaene) som en string, og det blir feil. Det er uansett ikke så mange identer per pers (blir overrasket om det er flere enn 3 på mange av tilfellene), så det er ikke kritisk kanskje at vi gjør en et batch-søk på identer i stedet for å kjøre det i en loop?